yt_dlp/extractor/youtube.py

```diff
@@ -68,14 +69,9 @@
     update_url_query,
     url_or_none,
     urljoin,
-    variadic,
+    variadic
```

Comment:
😕

Reply:
think that was my IDE when auto-refactoring :laughing: